docs/_layouts/default.html

```diff
@@ -9,7 +9,12 @@
     <meta name="description" content="{{ site.description }}">
 
     <link rel="stylesheet" href="{{ "/css/main.css" | prepend: site.baseurl }}">
-    <link rel="stylesheet" href="{{ "/css/default-theme.css" | prepend: site.baseurl}}" />
+    {% if site.env == 'development' %}
+      <link rel="stylesheet" href="http://localhost:8080/instantsearch.css" />
```

Comment:
The only difference with the js loading is that in development for example @Shipow that is not running `npm run dev` will not be able to see the default css file.

I think this is part of a bigger issue where the development mode of the docs is dependant of those files and thos files might not exists.

I dunno what we can do here, any idea?